biz.aQute.bndlib/bnd.bnd

```diff
@@ -23,6 +23,7 @@ Export-Package: \
 	aQute.bnd.service.*;-noimport:=true,\
 	aQute.bnd.help.*;-noimport:=true,\
 	aQute.bnd.header;-noimport:=true,\
+	aQute.bnd.home,\
```

Comment:
I think this should be `-noimport:=true` like the other exports.

Reply:
Yes, I think that is a good idea.

Reply:
done